mobile/src/main/java/org/openhab/habdroid/ui/widget/WidgetImageView.kt

```diff
@@ -148,9 +148,9 @@ class WidgetImageView constructor(context: Context, attrs: AttributeSet?) : AppC
         scope = null
     }
 
-    fun setRefreshRate(msec: Int) {
+    fun setRefreshRate(sec: Int) {
```

Comment:
This seems off, as the argument is given in milliseconds ... maybe name it `ms`?

Reply:
`ms` is better, I didn't know what `msec` means.